/**:
  ros__parameters:
    serial_port: /dev/ttyUSB-arduino-thruster
    serial_speed: 115200
    max_thrust: 80 # 全スラスタの最大合計推力
    max_thrust_per: 30 # スラスタ1つあたりの最大推力
